workers/unity/Packages/com.improbable.gdk.core/Config/ScriptableWorkerConfiguration.cs

```diff
@@ -5,7 +5,7 @@ namespace Improbable.Gdk.Core
 {
     public class ScriptableWorkerConfiguration : ScriptableObject
     {
-        public static readonly string AssetPath = "Assets/Gdk/Core/Config/ScriptableWorkerConfiguration.asset";
+        public static readonly string AssetPath = "Assets/ScriptableWorkerConfiguration.asset";
```

Comment:
Think this should be in `Assets/Config/...` rather than in the Asset root directory.

Or potentially a spatial directory

Reply:
Done